rich/progress.py

```diff
@@ -456,10 +456,13 @@ def __enter__(self) -> "Progress":
             return self
 
     def __exit__(self, exc_type, exc_val, exc_tb) -> None:
+        stopping = False
         with self._lock:
```

Comment:
I'm thinking there may not be much point in allowing a re-entrant context manager. Can't see any reason why it might be useful now...